src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -172,43 +174,47 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       }
     ];
 
+    const currentPermType = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
```

Comment:
The name of the variable sounds like it's a type but actually it's a boolean. Could be `isCurrentPermType` or some name more descriptive :)

Reply:
e2fd2f58af85cbb3116aebbe92abcc741779842f